components/tooltip.js

```diff
@@ -57,9 +59,16 @@ class Tooltip extends Component {
 
 Tooltip.propTypes = {
   children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired,
-  disabled: PropTypes.bool.isRequired,
+  disabled: PropTypes.bool,
+  width: PropTypes.number,
   tooltipText: PropTypes.string,
   className: PropTypes.string
 };
 
+Tooltip.defaultProps = {
+  disabled: false,
+  width: 150,
+  tooltipText: "tooltip text"
```

Comment:
Why not make tooltipText required with no default? Seems like using the default value here would always be a bug in any use of the component.

Reply:
good point! I'll change that.